eip-0023.md

```diff
@@ -0,0 +1,584 @@
+# Oracle Pool v2.0 
+
+* Author: @scalahub, @greenhat
```

Comment:
```suggestion
* Author: @scalahub, @greenhat, @kettlebell, @SethDusek
```
Since most of my requests for updates lately actually came from @kettlebell and @SethDusek.